piq/kid.py

```diff
@@ -79,8 +79,8 @@ def _mmd2_and_variance(
         diag_X = torch.diagonal(K_XX)
         diag_Y = torch.diagonal(K_YY)
 
-        sum_diag_X = diag_X.sum()
-        sum_diag_Y = diag_Y.sum()
+        sum_diag_X = torch.sum(diag_X)
+        sum_diag_Y = torch.sum(diag_Y)
```

Comment:
I am ok with this change but what is the difference?

Reply:
`torch.sum()` has correct typing info. `sum()` doesn't and throws an error

Reply:
Nice to know. Thanks 😄